orbit/pkg/service/service_notwindows.go

```diff
@@ -0,0 +1,10 @@
+//go:build !windows
+// +build !windows
+
+package service
+
+// This is currently a placeholder for non-windows OSes
+// system service configuration
+func SetupServiceManagement(serviceName string, serviceRootDir string) error {
```

Comment:
heads-up: the signature of this function is different from the one in `service_windows` (this one returns an error)

Reply:
good catch! fixed